exporters/trace/stackdriver/README.md

```diff
@@ -93,6 +93,14 @@ If you prefer to manually set the project ID use:
 StackdriverTraceExporter.createAndRegisterWithProjectId("MyStackdriverProjectId");
 ```
 
+#### Enable Stackdriver Trace API access scope on Google Cloud Platform
+
```

Comment:
Remove this empty line because it is not needed. Or add it everywhere.

Reply:
Done.